### ISSUE TITLE
Flags are missing

### ISSUE THREAD

**Issue by illwieckz** [XQF/xqf#156]
PR #147 from @skybon revealed that the flags were lost since 2015-02-10 because the autotools stuff fooled us.

**Comment by illwieckz** [comment]
Flags are stored in `pixmaps/flags`, if you think about a better place, feel free to suggest it.

**Comment by illwieckz** [comment]
For information, the Teeworld project have more than 240 flags (XQF only knows about 100) and they are nice:

* https://github.com/teeworlds/teeworlds/tree/master/datasrc/countryflags

**Comment by illwieckz** [comment]
I've just done that on my ubuntu box to test teeworlds flags in xqf:

```
(cd /usr/share/games/teeworlds/data/countryflags/; ls [A-Z][a-Z].png [A-Z][A-Z][a-Z].png | while read upName; do lowName="$(echo "${upName}" | tr '[:upper:]' '[:lower:]')"; echo convert "/usr/share/games/teeworlds/data/countryflags/${upName}" to "/usr/share/xqf/default/flags/${lowName}"; sudo convert "/usr/share/games/teeworlds/data/countryflags/${upName}" -resize "14x14^" -gravity center -crop 22x14+0+0 +repage "/usr/share/xqf/default/flags/${lowName}"; done)
```

It gives me that:

![XQF with Teeworlds country flags](http://dl.illwieckz.net/u/thomas-debesse/bug-reports/xqf/156-country-flags/20150622-072209.xqf-with-teeworlds-country-flags.png)

For information, by default we get that:

![XQF with default country flags](http://dl.illwieckz.net/u/thomas-debesse/bug-reports/xqf/156-country-flags/20150622-072724.xqf-with-default-country-flags.png)

The default flags are sharper but they have different sizes.

**Comment by illwieckz** [comment]
https://github.com/teeworlds/teeworlds/blob/master/license.txt said:

> All content under 'data' except the font (which has its own license) is
released under CC-BY-SA 3.0 (http://creativecommons.org/licenses/by-sa/3.0/).
